helpers/middlewares.js

```diff
@@ -0,0 +1,23 @@
+exports.processingMiddleware = async function processingMiddleware(err, req, res, next) {
+    if (!err.contextId) {
+        err.contextId = req.query.contextId;
+        err.pageId = req.query.pageId;
+    }
```

Comment:
I believe you don't need extra middleware for that, so you can move this logic to exceptionMiddleware.

If you'd like to have two middlewares, please rename processingMiddleware. One can guess what exceptionMiddleware does, but what does processingMiddleware process?